lib/views/login/login_form.dart

```diff
@@ -51,7 +51,7 @@ class LoginFormState extends ConsumerState<LoginForm> {
         ),
       ),
       onPressed: () {
-        context.push(RoutePath.forgetPassword.screen);
+        context.push(RoutePath.forgetPassword.path);
```

Comment:
we should use `context.pushNamed(RoutePath.forgetPassword.name)` after updating the route's structure

Reply:
You're right 👍